docs/src/howto/livegraphics.md

```diff
@@ -1,4 +1,4 @@
-# Interactive graphics and snapshots
+# Interactive graphics and Threads
```

Comment:
Perhaps

# Threads 

... something about ordinary drawings and threads etc for most existing users

# Interactivity and threads

... stuff about MiniFB and threads for people who want this new stuff ....